ekncontent/ekncontent/eknc-domain.c

```diff
@@ -891,7 +1022,7 @@ eknc_domain_query_finish (EkncDomain *self,
   QueryState *state = g_task_get_task_data (task);
   return g_object_new (EKNC_TYPE_QUERY_RESULTS,
                        "upper-bound", state->upper_bound,
-                       "models", state->models,
+                       "models", ptr_array_to_list_deep (state->models),
```

Comment:
No need to move it into a separate patch — this is apparently the only API that works with gobject-introspection :-)